code/math/curve.cpp

```diff
@@ -9,6 +9,61 @@ int curve_get_by_name(const SCP_string& in_name) {
 	return find_item_with_field(Curves, &Curve::name, in_name);
 }
 
+static int curve_inline_def(){
+	//The curve needs a unique identifier. Build this with semicolon + filename + semicolon + character in file
+	SCP_string curve_name = ';' + SCP_string(Current_filename) + ';' + std::to_string(Mp - Parse_text);
```

Comment:
Is there ever a possibility where curve names might need to be written to files?  If so, the semicolon would cause the rest of the line to be ignored.

Reply:
No.
In fact, this is why I added the semicolon. To make sure that there never is any naming conflict with tabled curves.